script/upgrades/MU03/MU03.sol

```diff
@@ -0,0 +1,440 @@
+    if (breakerBox != address(0) || BreakerBox(breakerBox).breakerTradingMode(medianDeltaBreaker) == 0) {
+      transactions.push(
+        ICeloGovernance.Transaction(
+          0,
+          breakerBox,
+          abi.encodeWithSelector(BreakerBox(0).addBreaker.selector, medianDeltaBreaker, 3)
+        )
+      );
+    }
+
+    // Add the Value Delta Breaker to the breaker box with the trading mode '3' -> trading halted
+    if (breakerBox != address(0) || BreakerBox(breakerBox).breakerTradingMode(valueDeltaBreaker) == 0) {
+      transactions.push(
+        ICeloGovernance.Transaction(
+          0,
+          breakerBox,
+          abi.encodeWithSelector(BreakerBox(0).addBreaker.selector, valueDeltaBreaker, 3)
+        )
+      );
+    }
+
+    // Set rate feed dependency for cEUR/USDC to cUSD/USDC.
+    transactions.push(
+      ICeloGovernance.Transaction(
+        0,
+        breakerBox,
+        abi.encodeWithSelector(
+          BreakerBox(0).setRateFeedDependencies.selector,
+          cEURUSDCConfig.referenceRateFeedID,
+          Arrays.addresses(cUSDUSDCConfig.referenceRateFeedID)
+        )
+      )
+    );
+
+    // Set rate feed dependency for cBRL/USDC to cUSD/USDC.
+    transactions.push(
+      ICeloGovernance.Transaction(
+        0,
+        breakerBox,
+        abi.encodeWithSelector(
+          BreakerBox(0).setRateFeedDependencies.selector,
+          cBRLUSDCConfig.referenceRateFeedID,
+          Arrays.addresses(cUSDUSDCConfig.referenceRateFeedID)
+        )
+      )
+    );
+
+    // Enable Median Delta Breaker for rate feeds
+    for (uint256 i = 0; i < poolConfigs.length; i++) {
+      if (poolConfigs[i].isMedianDeltaBreakerEnabled) {
+        transactions.push(
+          ICeloGovernance.Transaction(
+            0,
+            breakerBox,
+            abi.encodeWithSelector(
+              BreakerBox(0).toggleBreaker.selector,
+              medianDeltaBreaker,
+              poolConfigs[i].referenceRateFeedID,
+              true
+            )
+          )
+        );
+      }
+    }
+
+    // Enable Value Delta Breaker for rate feeds
+    for (uint256 i = 0; i < poolConfigs.length; i++) {
+      if (poolConfigs[i].isValueDeltaBreakerEnabled) {
+        transactions.push(
+          ICeloGovernance.Transaction(
+            0,
+            breakerBox,
+            abi.encodeWithSelector(
+              BreakerBox(0).toggleBreaker.selector,
+              valueDeltaBreaker,
+              poolConfigs[i].referenceRateFeedID,
+              true
+            )
+          )
+        );
+      }
+    }
+
+    // Set BreakerBox address in SortedOracles
+    transactions.push(
+      ICeloGovernance.Transaction(
+        0,
+        contracts.celoRegistry("SortedOracles"),
+        abi.encodeWithSelector(SortedOracles(0).setBreakerBox.selector, breakerBox)
+      )
+    );
+  }
+
+  function proposal_configureMedianDeltaBreaker() public {
```

Comment:
based on the config we are also enabling the median delta breaker for the cBRL/USDC and cEUR/USDC but I think you are only setting the cooldown and threshold for these rateFeeds `Arrays.addresses(cUSD, cEUR, cBRL)`

Reply:
yes, when I initially worked on the ticket the medianDeltaBreaker was only configured for these pools and they weren't updated since. I'll change them now!

Reply:
also noticed that valueDeltaBreaker was not configured for the new pools